scheduler/src/cook/mesos/scheduler.clj

```diff
@@ -200,7 +200,7 @@
 
 (defn handle-status-update
   "Takes a status update from mesos."
-  [conn driver ^TaskScheduler fenzo sync-agent-sandboxes-fn status]
+  [conn driver ^TaskScheduler pool->fenzo sync-agent-sandboxes-fn status]
```

Comment:
I think this type annotation is wrong now, can you move It down to the `fenzo` variable in the let block?

Reply:
Good call, will do

Reply:
Done